tools/read-dependencies

```diff
@@ -371,15 +387,7 @@ def pkg_install(pkg_list, distro, test_distro=False, dry_run=False):
         subprocess.call(
             ["dnf", "config-manager", "--set-disabled", "epel-cisco-openh264"]
         )
-    if distro in [
-        "suse",
-        "opensuse",
-        "fedora",
-        "redhat",
-        "rocky",
-        "centos",
-        "eurolinux",
-    ]:
+    if DISTRO_PKG_TYPE_MAP.get(distro) in ["redhat", "suse", "fedora"]:
```

Comment:
I think that this change slipped into the wrong commit, isn't this part of c8c97567dd59049e8d0b7c2ccabfadc11e32a4b6, rather than a suse-specific thing?